src/matter/interaction/InteractionErrors.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {MatterError} from "../../error/MatterError";
+import {StatusCode} from "./InteractionMessages";
+
+/** Specific Error for when a fabric is not found. */
+export class FabricNotFoundError extends MatterError {}
```

Comment:
You should move this in FabricManager.ts since this is thrown by it and might be handled in other code path unrelated to the InteractionProtocol.